docs/index.md

```diff
@@ -37,7 +37,7 @@ from
 
 ## Documentation
 
-- **[Table definitions & examples →](/plugins/turbot/heroku/tables)**
+- **[Table definitions & examples →](https://hub.steampipe.io/plugins/turbot/heroku/tables)**
```

Comment:
When rendering this doc, relative links are ok, so no need to add the `https://hub.steampipe.io` here